modules/Core/includes/admin_widgets/discord.php

```diff
@@ -21,21 +21,60 @@
         $discord_id = $discord_id[0]->id;
 
         if(isset($_POST['discord_api_key'])){
-            $discord_api_key = $_POST['discord_api_key'];
 
+            $validate = new Validate();
+            $validation = $validate->check($_POST, array(
+                'discord_api_key' => array(
+                    'min' => 18,
+                    'max' => 18,
+                    'numeric' => true
+                )
+            ));
+            
+            if ($validation->passed()) {
+                $discord_api_key = $_POST['discord_api_key'];
+            } else {
+                // Validation errors
+                foreach ($validation->errors() as $validation_error) {
+                    if (strpos($validation_error, 'minimum') !== false) {
+                        // x must be a minimum of y characters long
+                        switch ($validation_error) {
+                            case (strpos($validation_error, 'discord_api_key') !== false):
+                                $errors[] = str_replace('discord_api_key', 'Discord Server ID', $validation_error);
```

Comment:
Does this need translation support?

Reply:
I left it with no translations since other validations didn't have translations. What do you think?